README.md

```diff
@@ -63,6 +63,8 @@ Documentation of the LSST Science Pipelines is at https://pipelines.lsst.io
         $ ingestImagesDecam.py /path/to/repo --filetype raw /path/to/raw/*.fits.fz
         $ ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ --calibType defect /path/to/calib/*fits --validity 999
```

Comment:
Should the suggestion of using `--mode=link` go here (line 64) directly? Or are `cpBias`/`cpFlat` ingested differently than `defect`/`fringe`?  (Related to some questions below)